### PR TITLE
Fix suma API URL (master)

### PR DESCRIPTION
**Pull request by keichwa**
# Description
orthography normalization (drive-by edit)
Also see https://github.com/uyuni-project/uyuni-docs/pull/960

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/14903

### CODE REVIEW

**Review by juliogonzalez — APPROVED** [review]
LGTM. Before merging, verify if we have a link to the API somewhere else, just in case :-)

File: .changelog [hunk 1]
```diff
@@ -4,46 +4,6 @@
...
-- Adds instructions to delete vendor channels in
-  Administration Guide (bsc#1183293)
-- Adds troubleshooting entry to Administration Guide for
-  registering traditional RHEL 7 clients (bsc#1182700)
-- Adds missing Virtualization Network tab to System Details
-  section in Reference Guide
-- Adds SLE Micro and openSUSE MicroOS clients in the Client Configuration Guide
-- Adds troubleshooting multiple Red Hat CDN certificates to Client
-  Configuration Guide (bsc#1184448)
+- Fixed URL of API documentation
```
Comment (juliogonzalez) [inline .changelog:9]: Sorry, not OK, you are missing the bsc :-)

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: .changelog (hunk 1 above)
Comment (keichwa) [reply .changelog:9]: bsc for the API URL?  there is non, afaics.  Or is it about the other changelog entries?  We have the (bad) habit to reset the .changelog file when everything is transferred to the .changes of the package(s).  But I do not mind keeping the complete .changelog here.

Comment (juliogonzalez) [reply .changelog:9]: Ah, crap. I only created a github card :-D

Ignore my comment.

**Review by juliogonzalez — APPROVED** [review]
As long as you checked that this the only place where we link the API doc, LGTM.

File: .changelog (hunk 1 above)
Comment (keichwa) [reply .changelog:9]: ok ;)

**Pull request merged**